src/main/java/net/minecraftforge/registries/GameData.java

```diff
@@ -199,7 +199,7 @@ public static void init()
         return new RegistryBuilder<T>().setName(key.location()).setType(type).setMaxID(MAX_VARINT).hasWrapper().setDefaultKey(new ResourceLocation(_default));
     }
 
-    public static <T extends IForgeRegistryEntry<T>> MappedRegistry<T> getWrapper(ResourceKey<? extends Registry<T>> key, Lifecycle lifecycle)
+    public static <T extends IForgeRegistryEntry<T>> MappedRegistry<T> getWrapper(ResourceKey<? extends Registry<T>> key)
```

Comment:
Honestly, these changes are not needed. Yes they arguments are unused, but these are ease of use internal functions for our patches. Not something modders should be calling.